modules/mobile/docs/schemas/PreneedBurial.yml

```diff
@@ -0,0 +1,37 @@
+type: object
+required:
+  - data
+properties:
+  data:
+    type: array
+    items:
+      required:
+        - id
+        - type
+        - attributes
+      properties:
+        id:
+          type: string
+          example: "915"
+        type:
+          type: string
+          example: "cemetery"
+        attributes:
+          type: object
+          required:
+            - name
+            - type
+          properties:
+            name:
+              type: string
+              example: "ABRAHAM LINCOLN NATIONAL CEMETERY"
+            type:
+              type: string
+              enum:
+                - N
+                - S
+                - I
+                - A
+                - M
+              description: 'N=NATIONAL, S=STATE, I=INTERIOR, A=ARMY, P=PRIVATE, M=MILITARY'
```

Comment:
I talked with owners of this service and they said we shouldn't need to differentiate between the different types of cemeteries for the purposes of Preneed burial. So I included it in case we ever did and put the mapping in the docs for what each letter means but don't think it'll be necessary to show the mapping in the serialized data.